rmgpy/molecule/atomtype.py

```diff
@@ -370,7 +370,7 @@ def getFeatures(self):
 atomTypes['Cs'  ].setActions(incrementBond=['Cd','CO','Cs'], decrementBond=[],               formBond=['Cs'],        breakBond=['Cs'],        incrementRadical=['Cs'],   decrementRadical=['Cs'],   incrementLonePair=['Cs'],      decrementLonePair=['Cs'])
```

Comment:
Yeah it looks like just a typo to me. It certainly shouldn't be incrementing to Cs.